src/js/verkleFFIBindings/getTreeKey.ts

```diff
@@ -0,0 +1,115 @@
+import { Context } from '../../../dist/cjs/wasm/rust_verkle_wasm'
+
+// This is equal to 2n + 256n * 64n.
+//
+// It is a constant that is used in the `getTreeKeyHashJs` function.
+// See the `getTreeKeyHashJs` function for more details.
+const firstChunk = new Uint8Array([2, 64, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0])
+
+// Implements `get_tree_key` as specified here: https://notes.ethereum.org/@vbuterin/verkle_tree_eip#Header-values
+export function getTreeKeyJs(
```

Comment:
Hi there,
thanks a lot for all the work! Can we name this simply `getTreeKey()` and so without the `Js`? Or is there some specific reason to give this some "language appendix" I am not fully grasping on first sight?

Reply:
Oh this was just to differentiate between getTreeKey being implemented in Javascript versus being implemented in Rust. I'll remove it

Reply:
Ah, thanks, yeah, we are trying to name consistently in first round with the end user of the library in mind, and from an API usage perspective the Rust code is not visible/accessible anyhow any more.